content/api/customer-alerts.apib

```diff
@@ -0,0 +1,314 @@
+FORMAT: 1A
+title: Customer-Alerts
+description: Configure real-time alerts for your SparkPost account.
+
+# Group Alerts
+Alerts can be used to get notifications regarding various activities and metrics on your SparkPost account.
+
+Alerts can be configured to send notifications via email to up to 10 email addresses, and supports the following metrics:
+* Real-Time
+    + Monthly Sending limit
+* Signals
+    + Health Score
+
+Alerts are configured to be account-wide for Monthly Sending Limit.
+
+## Metric Limitations
+
+For each metric group, some configuration options simply don't make sense. The following metrics have restrictions:
+
+* Real-Time
+    + interval - Must be '15 Minutes'
```

Comment:
We should not expose interval in the API

Reply:
Corrected.